R/execution.r

```diff
@@ -78,11 +78,17 @@ Executor <- setRefClass(
 is.silent = function() {
     current_request$content$silent
 },
+is.store_history = function() {
+    current_request$content$store_history
+},
+request_metadata.get = function() {
```

Comment:
maybe just underscores if we want to switch to them for everything.

PS: in the options, the mixing of underscores and dots exists, because we use `jupyter.` to mean a “namespace” for the options.